internal/app/machined/main.go

```diff
@@ -241,6 +242,25 @@ func run() error {
 		log.Printf("controller runtime finished")
 	}()
 
+	var eg *errgroup.Group
+
+	eg, ctx = errgroup.WithContext(ctx)
+
+	eg.Go(func() error {
+		svc := &services.Containerd{}
+		if err = system.WaitForService(system.StateEventUp, svc.ID(c.Runtime())).Wait(ctx); err != nil {
```

Comment:
I believe `apid` service has a dependency on the `containerd` service already

Also I feel like we can start both after the `initialize` sequence, starting before `initialize` makes little sense to me. 

I also believe we don't need `errgroup` here